proxy/src/auth/backend.rs

```diff
@@ -276,11 +275,12 @@ async fn authenticate_with_secret(
     // Perform cleartext auth if we're allowed to do that.
     // Currently, we use it for websocket connections (latency).
     if allow_cleartext {
-        return hacks::authenticate_cleartext(info, client, &mut ctx.latency_timer, secret).await;
+        ctx.set_auth_method(crate::context::AuthMethod::Cleartext);
```

Comment:
We also need to add it to serverless/backend.rs::authenticate (or just to http-flow)

Reply:
yep. good point. don't make PRs tired on a Friday evening...